pkg/openthread/contrib/sock/ip/lwip_sock_ip.c

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
I temporally removed the ip folder because it was just a copy ofor using as template.